src/Custom-sample.h

```diff
@@ -427,23 +427,23 @@ static const char DATA_ESPEASY_DEFAULT_MIN_CSS[] PROGMEM = {
 // #define USES_P073   // 7-segment display
 // #define USES_P074   // TSL2591
 // #define USES_P075   // Nextion
-// #define USES_P076   // HLW8012/BL0937
+// #define USES_P076   // HLW8012/BL0937 (Sonoff POW Rxx)
```

Comment:
No I meant which one is used in the R1 and which in the R2 :)

Reply:
It seems HLW8032 is used on Sonoff POW R1 and CSE7766 is used on Sonoff S31 and Sonoff POW R2.
But I cannot see any reason to specify this in this file.

Reply:
This is the Custom.h file, so it is only required to be read/used by someone who wants to make their own builds.
So for this specific use case, it is very practical to know which one is used in which unit you have in your hands without doing a search or trying to decypher the chips on the PCB.

It is just some extra help.
No need to have these strings in the plugin code itself, but it is very useful to have when making your own builds.

Reply:
OK so for example also for SHT3x should be more info providing all the chips supported.
And I don't see this so detailed infos for the others comments.

Reply:
And IMHO, if someone use this file means that is enough "skilled" on ESPEasy.

Reply:
If the SHT3x is being used in some specific of the shelve products, then yes this would also qualify for such extra info.

Point I'm trying to make is that especially in the Custom.h it is useful to have such extra identifying info present to help those who want to build their own.

The people from Tasmota even go a step further, by having lots and lots of of the shelve devices being documented including some "setup string" which even configures Tasmota to be used on such a device.
But that's a completely different stage as Tasmota doesn't have so many different builds warranting people to make their own builds.

Reply:
It's OK if I add all these divices?
![image](https://github.com/letscontrolit/ESPEasy/assets/3474082/ecc9de7f-8f71-497d-a756-564755e3c69f)

Reply:
Sure, will be useful :)